server/pulp/server/webservices/views/content.py

```diff
@@ -268,6 +283,60 @@ def get(self, request):
         return generate_json_response(collection)
 
 
+class ContentUnitSearch(search.SearchView):
+    """
+    Adds GET and POST searching for content units.
+    """
+    optional_fields = ['include_repos']
+    manager = content_query.ContentQueryManager()
+
+    @staticmethod
+    def _add_repo_memberships(units, type_id):
+        """
+        For a list of units, find what repos each is a member of, and add a list
+        of repo_ids to each unit.
+
+        :param units:   list of unit documents
+        :type  units:   list of dicts
+        :param type_id: content type id
+        :type  type_id: str
+        :return:    same list of units that was passed in, only for convenience.
+                    units are modified in-place
```

Comment:
Is it actually convenient to return them? It's more clear that they are modified in place if they are not returned IMO.